### DISCUSSION TITLE
Pregnant PBPK

### DISCUSSION THREAD

**Discussion by AbilashReddybathini** [Open-Systems-Pharmacology/Forum#1425] [Questions & Problems] [answered]
I am unable to run a pregnant model in MOBI, I am unable to run the simulation, and
I am getting an issue saying "Unable to resolve path start value for entity cyp3a4".

I also request to kindly provide a "video tutorial on how to build a pregnant pbpk model using Mobi this can be very useful"

Thank you
![pregnant PBPK](https://user-images.githubusercontent.com/121849202/210309319-f0669add-b66d-4497-bfcb-ac50b75f712c.jpg)

**Comment by StephanSchaller** [comment, marked as answer]
Dear @AbilashReddybathini , you can find a course on use of the pregnancy model here: https://learn-esqlabs.talentlms.com/
